fiftyone/utils/paddleocr.py

```
@@ -0,0 +1,412 @@
+fou.ensure_torch()
+
+logger = logging.getLogger(__name__)
+
+# The PP-OCRv6 safetensors weights are run through paddleocr's ``transformers``
+# engine (torch only, no paddlepaddle). That engine needs a transformers new
+# enough to expose ``AutoModelForTextRecognition`` and the PP-OCR model types;
+# 5.0.0 (the version pinned in some enterprise images) is too old.
+_MIN_TRANSFORMERS = "transformers>=5.13"
+
+
+def _ensure_paddleocr():
```

Ruff (0.15.21)

[warning] Missing return type annotation for private function `_ensure_paddleocr`

Add return type annotation: `None`

(ANN202)